pkg/config/config.go

```diff
@@ -142,6 +142,16 @@ type PluginConfig struct {
 	// Log line longer than the limit will be split into multiple lines. Non-positive
 	// value means no limit.
 	MaxContainerLogLineSize int `toml:"max_container_log_line_size" json:"maxContainerLogSize"`
+	// NoCgroup indicates to disable the cgroup support. (auto/true/false)
```

Comment:
1) Update https://github.com/containerd/cri/blob/master/docs/config.md;
2) Add explanation for "auto".

Please do this for all 3 options.